zypper-docker crashes if the options for the lp/lu commands are at the end

The following makes zypper-docker crash:

```
$ ./zypper-docker lu suma-docker-registry.suse.de/opensuse-old-alsa --bugzilla
```

Note that `--bugzilla` is at the end, and not between the command and the image.

Backtrace:

```
panic: runtime error: index out of range

goroutine 1 [running]:
main.fixArgsForZypper(0xc82000a240, 0x4, 0x4, 0x0, 0x0, 0x0)
        /home/mssola/Projects/golang/src/github.com/SUSE/zypper-docker/helpers.go:106 +0x44e
main.main()
        /home/mssola/Projects/golang/src/github.com/SUSE/zypper-docker/zypper-docker.go:35 +0x77

goroutine 17 [syscall, locked to thread]:
runtime.goexit()
        /home/mssola/Projects/go/src/runtime/asm_amd64.s:1745 +0x1

goroutine 5 [syscall]:
os/signal.loop()
        /home/mssola/Projects/go/src/os/signal/signal_unix.go:22 +0x18
created by os/signal.init.1
        /home/mssola/Projects/go/src/os/signal/signal_unix.go:28 +0x37

goroutine 6 [select, locked to thread]:
runtime.gopark(0x8c99e8, 0xc820032728, 0x81eca8, 0x6, 0x43d218, 0x2)
        /home/mssola/Projects/go/src/runtime/proc.go:185 +0x163
runtime.selectgoImpl(0xc820032728, 0x0, 0x18)
        /home/mssola/Projects/go/src/runtime/select.go:392 +0xa64
runtime.selectgo(0xc820032728)
        /home/mssola/Projects/go/src/runtime/select.go:212 +0x12
runtime.ensureSigM.func1()
        /home/mssola/Projects/go/src/runtime/signal1_unix.go:227 +0x354
runtime.goexit()
        /home/mssola/Projects/go/src/runtime/asm_amd64.s:1745 +0x1

goroutine 7 [runnable]:
main.listenSignals.func1(0xc8200145a0)
        /home/mssola/Projects/golang/src/github.com/SUSE/zypper-docker/signals.go:32
created by main.listenSignals
        /home/mssola/Projects/golang/src/github.com/SUSE/zypper-docker/signals.go:43 +0xa4
```

--- a/helpers.go
+++ b/helpers.go
@@ -104,7 +104,7 @@ func fixArgsForZypper(args []string) []string {
 				sanitizedArgs = append(sanitizedArgs, "")
 				special = true
 
-				if len(args) >= (pos+1) && args[pos+1] == "" {
+				if len(args) > (pos+1) && args[pos+1] == "" {
 					skip = true
 				}
 				break

--- a/spec/update_operations_spec.rb
+++ b/spec/update_operations_spec.rb
@@ -16,9 +16,20 @@ describe "update operations" do
     end
   end
 
-  it "lists updates of an image" do
-    output = Cheetah.run("zypper-docker", "lu", Settings::VULNERABLE_IMAGE, stdout: :capture)
-    expect(output).to include("alsa-utils")
+  context "listing updates" do
+    it "lists updates of an image" do
+      output = Cheetah.run("zypper-docker", "lu", Settings::VULNERABLE_IMAGE, stdout: :capture)
+      expect(output).to include("alsa-utils")
+    end
+
+    # See issue: https://github.com/SUSE/zypper-docker/issues/66
+    it "does not crash when the --bugzilla flag is set after the image" do
+      begin
+        output = Cheetah.run("zypper-docker", "lu", Settings::VULNERABLE_IMAGE, "--bugzilla", stdout: :capture)
+      rescue Cheetah::ExecutionFailed => e
+        expect(e.message).to include "failed with status 1: flag provided but not defined: -bugzilla."
+      end
+    end
   end
 
   context "applying updates" do

